src/navigation/checkout/Login.js

```diff
@@ -31,6 +31,14 @@ class Login extends Component {
 
     return (
       <Center flex={ 1 }>
+        <View style={{ paddingHorizontal: 40, width: '100%' }}>
+          <Button colorScheme="success" onPress={() => this.props.guestModeOn()}>
+            { this.props.t('CHECKOUT_AS_GUEST') }
```

Comment:
Guest checkout can be enabled/disabled per instance. 
The setting should be exposed by the [settings endpoint](https://github.com/coopcycle/coopcycle-web/blob/db50f39d25692ef0d33cb5cc426a870658ba69e1/src/Action/Settings.php), and the app should show this only if enabled.

Reply:
Ok, I'll add it. Thanks